src/GitHub.InlineReviews/Commands/IVsCommand.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace GitHub.InlineReviews.Commands
+{
+    /// <summary>
+    /// Represents a Visual Studio command that does not accept a parameter.
+    /// </summary>
+    public interface IVsCommand : IVsCommandBase
```

Comment:
Yeah, the menus support parameters but are not nicely typed. We can kill the menu types or extend/replace bits in them with these interfaces.